contracts/test/nft_market.test.ts

```diff
@@ -0,0 +1,159 @@
+import { $log } from '@tsed/logger';
+import { BigNumber } from 'bignumber.js';
+import { TezosToolkit, MichelsonMap } from '@taquito/taquito';
+
+import { bootstrap, TestTz } from './bootstrap-sandbox';
+import { Contract, address, bytes } from '../src/type-aliases';
+
+import {
+    originateNftFaucet,
+    MintNftParam,
+    originateFixedPriceTezSale
+} from '../src/nft-contracts';
+import {
+    BalanceOfRequest,
+    addOperator
+} from '../src/fa2-interface';
+import { originateInspector, queryBalances } from './fa2-balance-inspector';
+
+jest.setTimeout(180000); // 3 minutes
+
+describe.each([originateFixedPriceTezSale])
```

Comment:
Should it be `describe.each([originateFixedPriceTezSale, originateFixedPriceSale])`?

Reply:
I looked at the documentation and tried this out as well. It just runs the entire set of tests in the describe block twice. This isn't necessary I would presume, so I didn't write that.

Reply:
My understanding was that you had two copies of the market test because you wanted to run it for two different types of the market contracts.
Indeed, my original suggestion was confusing since it used `originateFixedPriceTezSale` twice, but
I guess your intend was to run the same tests twice, but with two different types of the market contract originated by `originateFixedPriceTezSale` and `originateFixedPriceSale`